assets/AppImageThirdPartyNotices.txt

```diff
@@ -6,7 +6,7 @@
 
 
 Copyright (c) 1991-2016 Unicode, Inc. All rights reserved.
-Distributed under the Terms of Use in http://www.unicode.org/copyright.html
+Distributed under the Terms of Use in https://www.unicode.org/copyright.html
```

Comment:
this is supposed to be the third party license as is.  You should make sure this change is also made in the third party license

Reply:
Issue created: #8689

Reply:
Reverted in 01f8f794d9cda64a7a43cd4f389164be760da4c6